code/datums/martial/krav_maga.dm

```diff
@@ -1,189 +1,148 @@
 /datum/martial_art/krav_maga
 	name = "Krav Maga"
 	id = MARTIALART_KRAVMAGA
-	var/datum/action/neck_chop/neckchop = new/datum/action/neck_chop()
-	var/datum/action/leg_sweep/legsweep = new/datum/action/leg_sweep()
-	var/datum/action/lung_punch/lungpunch = new/datum/action/lung_punch()
+	var/datum/action/krav_maga/neck_chop/neckchop = new/datum/action/krav_maga/neck_chop()
```

Comment:
Is the additional nesting necessary?

Reply:
It's not wholly, but it does save a few lines of duplicated code. I can remove it if you'd prefer they remain duplicated

Reply:
It should keep similar format to the other martial arts.

the icon and required mob state are minor enough that I'd say its not very useful

Reply:
fair enough, I can swap that back out